src/attributes/decorator.js

```diff
@@ -28,6 +28,17 @@ function attributesDecorator(schema, schemaOptions = {}) {
       }
     });
 
+    function buildStrictMode(constructorArgs){
+      let instance = new WrapperClass(constructorArgs);
+
+      let {valid, errors} = instance.validate();
+      if(!valid) throw new Error(errors.map((err) => err.message).join('. '));
+
+      return instance;
+    }
+
+    WrapperClass.buildStrictMode = buildStrictMode;
```

Comment:
I think we call it just `buildStrict`, what do you think? The `mode` word doesn't add too much value to the name.